From 09090de0313776f114d626bbba8485f08a3e886a Mon Sep 17 00:00:00 2001
From: Jack Neely <jjneely@ncsu.edu>
Date: Wed, 7 Nov 2012 17:00:37 -0500
Subject: [PATCH] systemd: Implement hooks for tuning the AFS client

This makes a generic way to modify the behavior of the AFS client
similar to the old style afs_post_init bash function.
---
 src/packaging/RedHat/openafs-client.service |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/packaging/RedHat/openafs-client.service b/src/packaging/RedHat/openafs-client.service
index a2348cb..29548ac 100644
--- a/src/packaging/RedHat/openafs-client.service
+++ b/src/packaging/RedHat/openafs-client.service
@@ -7,6 +7,7 @@ Type=forking
 EnvironmentFile=/etc/sysconfig/openafs
 ExecStartPre=/sbin/modprobe openafs
 ExecStart=/usr/sbin/afsd $AFSD_ARGS $CACHE $OPTIONS $CONFDIR
+ExecStartPost=/usr/libexec/openafs/posthooks.sh
 ExecStop=/bin/umount /afs
 ExecStop=/usr/sbin/afsd -shutdown
 ExecStop=/sbin/rmmod libafs
-- 
1.7.1

